build-system/tasks/jsify-css.js

```diff
@@ -45,6 +45,9 @@ cssnano = cssnano({
   mergeIdents: true,
   reduceIdents: false,
   zindex: false,
+  svgo: {
+    encode: true,
```

Comment:
Mmm good intuition, but I checked the build directory, and it looks like `postcss-svgo` is smart about double-encoding and only encodes when necessary. Given that, do you think it's best to convert all the encoded SVGs now, or split that out into a smaller task?